R/write_dwc.R

```diff
@@ -56,6 +56,8 @@ write_dwc <- function(connection = con,
     length(animal_project_code) == 1,
     msg = "`animal_project_code` must be a single value."
   )
+  ## Set animal project code to lowercase for sql
+  animal_project_code <- stringr::str_to_lower(animal_project_code)
```

Comment:
Are all project codes lowercase in the database then?

Reply:
I'm relying on the validity of this line in `get_animal_projects()`:

https://github.com/inbo/etn/blob/326d9e8293c51080b87a3b8569f6a3df7291225f/R/get_animal_projects.R#L39-L42

If it's ok to `LOWER()` all codes there, surely it's ok to do the same here?

Reply:
Oh, I see, they are lowered on the database side, and then compared to the client entry. It's not entirely the same. 

We might need a different solution then! How do we handle this for the other functions? We could change the sql for `write_dwc()` I suppose.

Reply:
I think it's fine. In the dwc_sql we compare with the lowered code too:

https://github.com/inbo/etn/blob/326d9e8293c51080b87a3b8569f6a3df7291225f/inst/sql/dwc_occurrence.sql#L16

https://github.com/inbo/etn/blob/326d9e8293c51080b87a3b8569f6a3df7291225f/inst/sql/dwc_occurrence.sql#L78

https://github.com/inbo/etn/blob/326d9e8293c51080b87a3b8569f6a3df7291225f/inst/sql/dwc_occurrence.sql#L95

Reply:
In that case I'll go ahead and merge. We can revisit it if someone finds an exception!